### ISSUE TITLE
build: only run generate-images target if node and index.js available (i.e. in docker)

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/music-encoding#1211]
otherwise a local build will fail

**Comment by musicEnfanthen** [comment]
There is already a coniditonal check in [line 290](https://github.com/music-encoding/music-encoding/blob/develop/build.xml#L290) ?

Perspectively, it will not require node anymore but python with #1184

**Comment by bwbohl** [comment]
Yes probably you are right, as the target will only be run if the docker property is set, which of course anyone can do at any time… but yes. Probably there should be a check for the python version of generating images, but for the moment I'd say we postpone this to 5.x

**Comment by bwbohl** [comment]
To make the target work in non-Docker contexts, we should check for the presence of the correct Python version and Verovio.
